### PR TITLE
Spelling and grammar fixes

### PR DESCRIPTION
**Pull request by itsnotrin**
This PR isn't done yet, however, I'm just adding it so that other people don't run off and do the same things.

### CODE REVIEW

**Review by VincentRPS — APPROVED** [review]
Current stuff LGTM, will need to approve again once done.

File: docs/Popular-Topics/sharding.mdx [hunk 1]
```diff
@@ -7,31 +7,31 @@ description: All about Sharding Discord Bots.
...
-just useless, but actually _harmful_, as it uses extra resources and may slow down your bot. Discord
-themselves will actually let you know when your bot needs sharding.
+your bot, it isn't necessary to shard unless your bot is large. Sharding a small bot isn't
+just useless, but _harmful_, as it uses extra resources and may slow down your bot. Discord
+themselves will let you know when your bot needs sharding so you won't need to worry about this until they contact you.
 
 ## How Do I Shard in Pycord?
 
-Sharding in Pycord is straight-forward.
+Sharding in Pycord is straightforward and extremely easy.
```
Comment (EmreTech) [inline docs/Popular-Topics/sharding.mdx:26]: This seems redundant.

File: docs/Popular-Topics/sharding.mdx [hunk 2]
```diff
@@ -7,31 +7,31 @@ description: All about Sharding Discord Bots.
...
 
 Just replace your `discord.Client`, `discord.Bot`, `commands.Bot` objects
 with `discord.AutoShardedClient`, `discord.AutoShardedBot`, `commands.AutoShardedBot` respectively.
 
-These new objects have inherit from the original objects, so all of the functions from those other types should be present. 
+These new objects have been inherited from the original objects, so all of the functions from those other types should be present.
```
Comment (EmreTech) [inline docs/Popular-Topics/sharding.mdx:31]:
```suggestion
These new objects are inherited from the original objects, so all of the functions from those other types should be present. 
```

File: docs/Popular-Topics/subclassing-bots.mdx [hunk 3]
```diff
@@ -5,9 +5,9 @@ description: Subclassing is a popular way of creating Discord Bots. Explore how
 
 ## About
 
-Subclassing is another popular way of creating Discord Bots. Here, you create a bot by extending a bot class. To some advanced users, this is more preferable than creating a bot instance with `bot = discord.Bot()`.
+Subclassing is another popular way of creating Discord Bots. Here, you create a bot by extending a bot class. To some advanced users, this is preferable to creating a bot instance with `bot = discord.Bot()`.
 
-Subclassing is an intermediate python concept, so we recommend you learn about it before continuing. Some good resources are:
+Subclassing is an intermediate python concept, so we commend you to learn about it before continuing. Some good resources are:
```
Comment (EmreTech) [inline docs/Popular-Topics/subclassing-bots.mdx:10]:
```suggestion
Subclassing is an intermediate python concept, so we recommend you to learn about it before continuing. Some good resources are:
```

**Review by EmreTech — DISMISSED** [review]
LGTM

**Pull request merged**